models/transfers/celo/erc20/transfers_celo_erc20.sql

```diff
@@ -0,0 +1,101 @@
+{{ 
+    config(
+        tags = ['dunesql'],
+        alias = alias('erc20'),
+        partition_by = ['block_month'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['block_time', 'tx_hash', 'index', 'wallet_address', 'amount_raw'],
+        post_hook='{{ expose_spells(\'["celo"]\',
+                                    "sector",
+                                    "transfers",
+                                    \'["soispoke", "dot2dotseurat", "tschubotz", "tomfutago"]\') }}'
+    )
+}}
+
+with
+    sent_transfers as (
+        select
+            to as wallet_address,
```

Comment:
Maybe we can add a column here:
```suggestion
        select
            'send' as type,
            to as wallet_address,
```

Reply:
Sure, I can do that, but before I go ahead - just want to check few things:

1) my initial attempt was aimed to replicate the same table structure as transfers for all other chains. With all changes I applied so far, transfers_celo (on both celo and erc20) does no longer match table schema for other chains - it that ok?

2) current error msg ("TrinoExternalError(type=EXTERNAL, name=DELTA_LAKE_BAD_WRITE, message="Unable to rewrite Parquet file"") shows for both transfers_celo_celo and transfers_celo_erc20, so I fail to see how adding a column to just erc20 table will address this error. Can you elaborate please?

Thanks!

Reply:
Also, not sure if that's relevant but I found these when trying to find anything about this error msg:

https://github.com/trinodb/trino/issues/16413
https://github.com/trinodb/trino/pull/18506

Does it help in any way?

Reply:
For the first question, I think it is ok(as we can update the corresponding schema).
Transfers tables are large tables, I was thinking the error may be related to their table size.

Reply:
Fair point, I wonder what would happen if transfers for any other chain got updated as celo is one with quite small dataset to load comparing to most of the ones already loaded on Dune.

Reply:
The daily incremental updates happen for other existed transfers on other chains, which is relatively smaller size. Once the spell is merged, it may update fine as well.

Reply:
I added cast block_month as date and that seem to do the trick!